File: Week_0/SeJun/3052.cpp
#include <iostream>

using namespace std;

int cnt[43];

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);

	int num;
	int result = 0;
	for (int i = 0; i < 10; i++) {
		cin >> num;
		cnt[num % 42]++;
	}
	for (int i = 0; i < 42; i++) {
		if (cnt[i] > 0) result++;
	}
	cout << result;
}

File: Week_0/JeongminKim/Base/3052.py
"""

    수 10개를 입력 받은 뒤, 이를 42로 나눈 나머지를 구한다.

"""
i=1
ls = []
while 1:
    if i > 10:
        break

    num = int(input()) % 42
    if num not in ls:
        ls.append(num)
    i += 1
print(len(ls))